### PR TITLE
APIS-6756 - Play 3.0 and use the ui-test-runner

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: tmp.txt
Comment (johnsgp) [inline tmp.txt:1]: I'm guessing this was not supposed to be committed.

File: run_all_tests.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 export SBT_OPTS="-XX:+CMSClassUnloadingEnabled -Xmx4G"
-sbt pre-commit
+sbt -Dwebdriver.chrome.driver=/usr/bin/chromedriver pre-commit
```
Comment (johnsgp) [inline run_all_tests.sh:3]: Is this intentionally using the local driver?

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -4,53 +4,54 @@ import play.sbt.PlayImport._
...
     "io.cucumber"                 %  "cucumber-junit"                     % cucumberVersion,
-    "io.cucumber"                 %  "cucumber-java8"                     % cucumberVersion,
-    "org.seleniumhq.selenium"     %  "selenium-remote-driver"             % seleniumVersion,
-    "org.seleniumhq.selenium"     %  "selenium-firefox-driver"            % seleniumVersion,
-    "org.seleniumhq.selenium"     %  "selenium-chrome-driver"             % seleniumVersion,
-    "org.scalatestplus"           %% "selenium-4-2"                       % "3.2.13.0",
-    "com.github.tomakehurst"      %  "wiremock-jre8-standalone"           % "2.33.2",
-    "uk.gov.hmrc"                 %% "webdriver-factory"                  % "0.38.0"
+    "com.titusfortner"     %  "selenium-logger"             % "2.3.0",
+    // "org.seleniumhq.selenium"     %  "selenium-remote-driver"             % seleniumVersion,
```
Comment (johnsgp) [inline project/AppDependencies.scala:51]: Can these comments be deleted?

File: tmp.txt
Comment (AndySpaven) [reply tmp.txt:1]: Oh I don't know.... ;-)

File: run_all_tests.sh (hunk 1 above)
Comment (AndySpaven) [reply run_all_tests.sh:3]: No - I'm confused it's still there as I worked out how to get it working using JAVA_TOOLS_OPTIONS so this was removed.  Maybe a late local commit that was not pushed.

File: project/AppDependencies.scala (hunk 2 above)
Comment (AndySpaven) [reply project/AppDependencies.scala:51]: Yes

**Comment by peteslater-ee** [comment]
test this please

**Comment by mattclark-zerogravit** [comment]
test this please

**Comment by peteslater-ee** [comment]
test this please

File: tmp.txt
[the remark this answers is not in the document]
Comment (AndySpaven) [reply tmp.txt:1]: Nope
Drat. Rebasing bit me on several things...

File: build.sbt [hunk 3]
```diff
@@ -72,9 +73,9 @@ lazy val microservice = Project(appName, file("."))
   .settings(inConfig(ComponentTest)(Defaults.testSettings ++ BloopDefaults.configSettings ++ ScalafmtPlugin.scalafmtConfigSettings) ++ scalafixConfigSettings(ComponentTest))
   .settings(headerSettings(ComponentTest) ++ automateHeaderSettings(ComponentTest))
   .settings(
-    ComponentTest / testOptions := Seq(Tests.Argument(TestFrameworks.ScalaTest, "-eT")),
+    // ComponentTest / testOptions := Seq(Tests.Argument(TestFrameworks.ScalaTest, "-eT")),
```
Comment (johnsgp) [inline build.sbt:76]: Another comment

**Review by johnsgp — DISMISSED** [review]

**Review by mattclark-zerogravit — APPROVED** [review]
👍

**Pull request merged**